python/ctsm/modify_fsurdat/fsurdat_modifier.py

```diff
@@ -0,0 +1,100 @@
+"""
+Run this code by using the following wrapper script:
+tools/modify_fsurdat/fsurdat_modifier
+
+The wrapper script includes a full description and instructions.
+"""
+
+#  Import libraries
+import sys
+from getpass import getuser
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+
+from ctsm.modify_fsurdat.modify_fsurdat import ModifyFsurdat
+
+
+def get_parser():
+
+    """
+    Get parser object for this script.
+
+    Command-line inputs
+    - fsurdat_in: input file (str)
+    - fsurdat_out: output file (str)
+    - variable: variable to modify (str)
+    - value: value assigned to the variable to be modified (float)
+    """
+
+    parser = ArgumentParser(description=__doc__,
+                           formatter_class=RawDescriptionHelpFormatter)
+
+    parser.print_usage = parser.print_help
+
+    parser.add_argument('--dom_pft',
+                help='Dominant PFT if overwrite_single_pft = .true. [default: %(default)s] ',
+                action="store",
+                dest="dom_pft",
+                type =int)
+    parser.add_argument('--uni_snow',
+                help='Turn on the flag to create uniform snowpack. [default: %(default)s]',
+                action="store_true",
+                dest="uni_snowpack")
+    parser.add_argument('--overwrite_single_pft',
+                help='Turn on flag to make whole grid 100% single PFT. [default: %(default)s]',
+                action="store",
+                dest="overwrite_single_pft",
+                default=False)
```

Comment:
- [ ] For this and a few other booleans, you could follow the pattern of setting `action="store_true"` and getting rid of the `default=False`. (You did that for a few but not all.)